class NotFoundException(Exception):
    pass


class HostNotFoundException(NotFoundException):
    pass
